crates/forge-runner/src/sierra_test_code_path.rs

```diff
@@ -0,0 +1,35 @@
+use crate::compiled_runnable::CompiledTestCrateRunnable;
+use anyhow::Context;
+use anyhow::Result;
+use cairo_lang_sierra::program::VersionedProgram;
+use camino::Utf8PathBuf;
+use fs4::FileExt;
+use std::fs::File;
+use std::io::BufWriter;
+
+pub struct SierraTestCodePath {
```

Comment:
I think `Code` is word which we don't need, how compiler refers to this structure?

Reply:
Changed to `Program` + changed the name a bit